pkg/iwebsocket/iwebsocket.go

```diff
@@ -0,0 +1,7 @@
+package iwebsocket
+
+type IWebSocket interface {
```

Comment:
As mentioned, both package and interface should drop the `i`

Reply:
done.